API/common/utils.py

```diff
@@ -41,7 +41,15 @@ def execute(cwd, cmd, args=None, quiet=False):
     stdout = None
     stderr = None
 
-    if quiet:
+    verbose = os.environ.get('VERBOSE', '')
+    jsverbose = os.environ.get('JSVERBOSE', '')
+
+    if verbose and not jsverbose:
+        console.warning('Please define JSVERBOSE environmental variable to enable verbose output')
```

Comment:
Ok, thanks, I understand now. This solution will print this message many times. You should move this check to `driver.py`.